doc/manual/manual.tex

```diff
@@ -2771,20 +2773,126 @@ \subsubsection{Compiling \aspect{} and generating documentation}
 result will be the file \url{doc/doxygen/index.html} that is the starting
 point for exploring the documentation.
 
-\subsubsection{Compiling a static \aspect{} executable}
-\aspect{} defaults to a dynamically linked executable, which saves disk space and build time. In some circumstances however, it is preferred to generate a statically linked executable that incorporates all used libraries. This need may arise on large clusters on which libraries and loaded modules/variables on login nodes may be different from the ones available on compute nodes. The general build procedure in such a case equals the above explained instructions with the following differences:
-\begin{enumerate}
-\item \textit{Trilinos}: Add the following lines to your cmake call:
+\subsubsection{Special cases}
+
+\paragraph{Installation without internet access:}
+
+\texttt{candi} relies on the fact that it can download packages from known
+internet locations. Some cluster systems might prevent internet access or
+disallow certain SSL certificates. In these cases please install \texttt{candi}
+and \dealii{} on a different system (e.g. a desktop system) and then copy the
+downloaded packages to your cluster. \texttt{candi} will recognize the
+existence of the packages and skip the download phase. The packages should be
+placed in the following folder: 
+
+\begin{verbatim} 
```

Comment:
no empty line here (and elsewhere) -- an empty line starts a new paragraph (and indents the text), but here and below you just want the current sentence or thought to continue.